### DISCUSSION TITLE
Some important tips/solutions to issues I've found

### DISCUSSION THREAD

**Discussion by erikwallindermahler** [weathondev/SingstarCreatorTutorial#4] [General]
Been playing around with the singstar-creator tool, and wanted to share some things that took me way too long to figure out:

* singstar-creator cannot use a #BPM containing a comma, but changing it to a period is fine. Unsure about commas in other spots in the txt files
* Important: singstar-creator generated unbootable iso:s when I started testing it (using 1-3 songs). This issue disappeared when generating iso:s with more songs, tested iso:s with 10 and 40 songs now that are working. Some medleys are also automatically created.
* If an error about a memory access violation happens when importing, there could be something wrong with the formatting of the text file. I had newlines everywhere in a file, removing them solved this. There are other similar errors that I'm not sure what causes them however, but they are only for specific songs.

Right now I'm testing patching the iso and putting it on DVD:s to attempt running it on unmodded hardware, hopefully it works! Currently possible to run from disc in PCSX2 at least. Great guide, would have been impossible to get anywhere without it!

**Comment by Brawdogger** [comment]
Hey, I noticed you were able to generate Iso's with 40 songs. Did you ever have an issue where singstar creator would get stuck during the conversion of txt files and what did you do to fix it? Seems to be an issue that is plaguing a few of us trying to use the tool right now
